packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -869,21 +869,26 @@ Content-Type: application/json
+* If the new or unknown device email notification is enabled, an email is sent to the user if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. For more information about this feature, see the [General Security documentation](https://help.okta.com/en/prod/Content/Topics/Security/Security_General.htm?).
+* If you have the security behavior detection feature enabled and you have a new device behavior configured in a policy rule, a new device is detected if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. For more information about this feature, see [EA documentation](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm?).
 
-Specifying your own device fingerprint is a highly privileged operation limited to trusted web applications and requires making authentication requests with a valid API token.
-You should send the device fingerprint only if the trusted app has a computed fingerprint for the end user's client.
+Specifying your own device fingerprint in the `X-Device-Fingerprint` header is a highly privileged operation that is limited to trusted web applications and requires making authentication requests with a valid API token. You should send the device fingerprint only if the trusted app has a computed fingerprint for the end user's client.
 
-**Notes:**
+> **Note:** The `X-Device-Fingerprint` header is different from the device token. Device-based MFA in the Okta Sign-On policy rules depends on the device token only and not on the `X-Device-Fingerprint` header. To read more about the device token, see [Context Object](#context-object). Device-based MFA would work only if you pass the device token in the [client request context](/docs/reference/api-overview/#client-request-context).
 
-* Device fingerprint is different from the device token. The time and device based MFA in Okta SignOn policy rules depends on the device token only and not on the device fingerprint. To read more about the device token, see [Context Object](#context-object). The time and device based MFA would work only if you send the device token passed in the [client request context](/docs/reference/api-overview/#client-request-context).
-* To use device fingerprinting for the new or unknown-device email notification feature, include the `User-Agent` header in the request. For more information, see the [General Security documentation](https://help.okta.com/en/prod/Content/Topics/Security/Security_General.htm?).
+##### Device Fingerprint Best Practices
```

Comment:
This is confusing, we are adding the best practices for Device *Token* and saying it's the best practices for *Fingerprint*.

We should re-work this whole section to better describe the top-level use cases and then how they map to implementation details like Device Token vs Device Fingerprint. (I know we have some forthcoming changes to make the product work correctly, but we need to make the docs correct ahead of time and explain what's changing). I thought PM was going to help structure this, but here's how I thought it would look:

| Use Case | Features Available | Notes |
| --------- | ------------------- | ------ |
| Require MFA for users on unknown devices |  | Encourage admins to configure default (fall-through) rules to be secure (MFA) by default, and configure per-device rules to *opt-out* of MFA, not opt-in. |
| | Per-device option in App SignOn Policy | Based on Device Token |
| | Risk Score in Okta SignOn Policy | Based weighted combo of Device Token + Device Fingerprint + other signals |
| | New Device Behavior in Okta SignOn Policy (only for advanced mode; we should [discourage folks from using](https://oktawiki.atlassian.net/wiki/spaces/PM/pages/925597862/Rethinking+IP+based+security+behaviors) unless they have an advanced case that Risk Score doesn't address) | Based on Device Fingerprint. NOTE: Okta has determined that Device Fingerprint is not a strong proof of device. This feature will change to use Device Token in an upcoming release. |
| Send "New Device" notifications to users | New Sign In Notification Email (under Security->General) | Based on Device Fingerprint |

https://oktawiki.atlassian.net/wiki/spaces/PM/pages/734406988?focusedCommentId=903167566#comment-903167566

To this end, "Primary Authentication with Device Fingerprinting" really shouldn't be a first-class scenario like we're making it:

<img width="881" alt="Authentication" src="https://user-images.githubusercontent.com/6208297/77011961-fc4ac680-693a-11ea-9714-bba3e530e52c.png">

It should be "Primary Authentication with Known Device", and then we map this back to Use Case 1 in that table above.

Reply:
> We should re-work this whole section to better describe the top-level use cases and then how they map to implementation details like Device Token vs Device Fingerprint. (I know we have some forthcoming changes to make the product work correctly, but we need to make the docs correct ahead of time and explain what's changing)

I think it would be confusing to document forthcoming product changes (particularly while we are considering how to standardize header vs context body). In my opinion, the use cases and the notes you captured, while valid, should be documented when we [finalize the plan - I will setup the design meeting next week](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/916015568/Technical+Design+for+Improvements+to+New+Device+Behavior+Detection).

> This is confusing, we are adding the best practices for Device Token and saying it's the best practices for Fingerprint.

I think this is basically providing the same best practices that we did to the customer couple of weeks back. That customer was also sending a UUID through the X-Device-Fingerprint. Until we implement the proposal we were discussing to improve the behavior, we dont want customers to generate browser fingerprints using AuthJS and we want them to send UUIDs (that are securely stored in cookies). To that end, I think the way the doc is written out now is correct - It captures the best way to use the feature today. We will have to come back and update the doc again when we finalize/implement the proposal.